package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "devDependencies": {
+    "ts-node": "^10.9.1",
+    "typescript": "^5.1.6"
+  },
+  "dependencies": {
+    "@types/node": "^20.4.6",
+    "emojibase": "^15.0.0",
+    "emojibase-data": "^15.0.0",
+    "i": "^0.3.7",
+    "npm": "^9.8.1"
```

Comment:
We should be using `yarn` and not including a package manager as a dev dependency as it cause issues on dev boxes